encoding/ipfix/element.go

```diff
@@ -4,63 +4,88 @@ import (
 	"fmt"
 )
 
+const (
+	fieldSpecifierSize            = 4
+	fieldSpecifierWEnterpriseSize = 8
+)
+
+// ErrorElement is an error definition associated with IPFIX Elements.
```

Comment:
Codacy found an issue: [comment on exported type ElementError should be of the form "ElementError ..." (with optional leading article)](https://app.codacy.com/manual/mateumann/clouseau/pullRequest?prid=6315366)